meson.build

```diff
@@ -32,7 +32,12 @@ code = '''#ifndef __WINE__
 dxvk_winelib = dxvk_compiler.compiles(code, name: 'winelib check')
 dxvk_extradep = [ ]
 
+cpp_args = [ ]
+
 if dxvk_winelib
+  if dxvk_compiler.has_argument('--no-gnu-unique')
+    cpp_args += '--no-gnu-unique'
```

Comment:
Please just use `add_project_arguments` directly like in all the other places instead of introducing random new variables.

Reply:
Done.

This was done following the example in mesa's build and is more robust, but okay...